### PR TITLE
Edited Mentee's link in mentorship page table

### LINKED ISSUE #585: Adding a collapse section for all the contributors in the mentorship page
hey, @klumb I think this small enhancement can be helpful for the website, Please have a look!
Right now in the [mentorship section](https://chaoss.community/chaoss-mentorship/) of the website has a table listing all the contributors that have contributed to this organization through many programs, but the links to the individual contributor section in the table section will only work in Github markdown, but not in the website
For Example : 
> |[Ria Gupta](#Ria-Gupta)|2020|Social Currency Metric System|Valerio Cosentino, Samantha Venia, Logan|
above line of code will not interlink the [**#Ria-Gupta**](https://github.com/chaoss/website/blob/master/Community/alumni.md#ria-gupta) section on website, it will only add **#Ria-Gupta** to the suffix of the URL

### PR DESCRIPTION
**Pull request by robin025**
Fixes #585 
hey @GeorgLink  and @klumb 
actually I had a discussion with @klumb   on issue #585, so I have decided to add **#user-content** keyword before every mentee's link so that it can redirect user to that section

### CODE REVIEW

**Comment by klumb** [comment]
This looks good, but can you remove any edits to the CHAOSScon pages. I think they should be archived the way they are.

**Comment by robin025** [comment]
Actually @klumb I have already reverted all the changes in CHAOSScon pages that i have created and @GeorgLink  have merged that PR #590

**Review by vchrombie — CHANGES_REQUESTED** [review]
stumbled upon this

File: Community/alumni.md [hunk 1]
```diff
@@ -9,24 +9,24 @@ The CHAOSS Community thanks all mentees and mentors who participated in Google S
...
+|[Ria Gupta](#user-content-Ria-Gupta)|GSoC 2020|Social Currency Metric System|Valerio Cosentino, Samantha Venia, Logan|
+|[Tianyi Zhou](#user-content-Tianyi-Zhou)|GSoC 2020|Large Social Network Analysis and Anomaly Detection with Augur|Sean Goggins, Jonahz, Gabe Heim|
+|[Sarit Adhikari](#user-content-Sarit-Adhikari)|GSoC 2020|Machine Learning for Anomaly Detection in Open Source Communities|Sean Goggins, Carter Landis, Gabe Heim|
+|[Abhinav Bajpai](#user-content-Abhinav-Bajpai)|GSoC 2020|Implementing GitLab Data Collection Worker and Mapper to bind the responses of GitLab API, Github API & the Augur schema|Sean Goggins, Carter Landis|
+|[bistaastha](#user-content-bistaastha)|GSoC 2020|Build Workflow process for CHAOSS D&I Badging Project|Matt Snell|
+|[vchrombie](#user-content-vchrombie)|GSoC 2020|Creating Quality models using GrimoireLab and CHAOSS metrics|Valerio Cosentino, Aniruddha Karajgi|
+|[Saicharan Reddy](#Suser-content-aicharan-Reddy)|GSoC 2020|Implementation of GitLab Data Collection Worker & Test Coverage Improvement|Elita Nelson, Sean Goggins, Jonahz|
```
Comment (vchrombie) [inline Community/alumni.md:20]: oops, a typo

**Review by vchrombie — CHANGES_REQUESTED** [review]

File: Community/alumni.md (hunk 1 above)
Comment (robin025) [reply Community/alumni.md]: Done... @vchrombie, fixed the typo

**Comment by klumb** [comment]
Is this ready to merge? @vchrombie

**Comment by vchrombie** [comment]
Thanks @robin025 for the work.
Sorry @klumb, I missed the notification. Let me check it once again.

**Review by vchrombie — APPROVED** [review]
LGTM! I hope the changes would fix the scrolling issue.

**Comment by klumb** [comment]
Hi, @robin025 I merged this but it does not fix the scrolling issue.

For this to work, the URL link needs to be lower case.


For example:
[Jaskirat Singh](#user-content-jaskirat-singh)
https://chaoss.community/chaoss-mentorship/#user-content-jaskirat-singh

If you go back and make all the names in the link lower case it should work.

**Comment by robin025** [comment]
> Hi, @robin025 I merged this but it does not fix the scrolling issue.
> 
> For this to work, the URL link needs to be lower case.
> 
> For example:
> [Jaskirat Singh](#user-content-jaskirat-singh)
> https://chaoss.community/chaoss-mentorship/#user-content-jaskirat-singh
> 
> If you go back and make all the names in the link lower case it should work.


okay.. @klumb

**Pull request merged**